Xamarin.Essentials/Contacts/Contacts.ios.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Contacts;
+using ContactsUI;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Contacts
+    {
+        static Task<Contact> PlatformPickContactAsync()
+        {
+            var uiView = Platform.GetCurrentViewController();
+            if (uiView == null)
+                throw new ArgumentNullException($"The View Controller can't be null.");
+
+            var source = new TaskCompletionSource<Contact>();
+
+            using var picker = new CNContactPickerViewController
```

Comment:
How does this work with the GC? I recall somewhere that iOS may collect this instance, and thus close the popup.
Do we need to keep a reference to it at all?